### PR TITLE
Create OEM diag workflow

### PR DESCRIPTION
**Pull request by jlongever**
@RackHD/corecommitters @tannoa2 @uppalk1 @keedya @zyoung51 

Dependency on PR https://github.com/RackHD/on-http/pull/345 and  RackHD/on-tasks#256

### CODE REVIEW

**Comment by benbp** [comment]
Depends on https://github.com/RackHD/on-tasks/pull/256 (on-http PR linked twice).

**Comment by jlongever** [comment]
@benbp thanks.

File: lib/graphs/run-emc-diag-graph.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Run EMC Diagnostics',
+    injectableName: 'Graph.Run.Emc.Diag',
+    options: {
+        'bootstrap-emc-diag': {
+            'completionPath': '/api/common/templates/{{ options.completionUri }}'
```
Comment (benbp) [inline lib/graphs/run-emc-diag-graph.js]: You can do `{{ api.templates }}/{{ options.completionUri }}` too if you so desire :)

**Comment by benbp** [comment]
👍

File: lib/graphs/run-emc-diag-graph.js (hunk 1 above)
Comment (jlongever) [inline lib/graphs/run-emc-diag-graph.js]: Thanks.

**Comment by jlongever** [comment]
jenkins, test this please

**Pull request merged**